zap/src/main/java/org/parosproxy/paros/CommandLine.java

```diff
@@ -541,7 +541,7 @@ public boolean isNoStdOutLog() {
     }
 
     /**
-     * Returns true if ZAP should not make any unsolicited requests, eg check-for-updates etc
+     * Returns true if ZAP should not make any unsolicited requests, e.g. check-for-updates etc
```

Comment:
This `etc` does not have a dot, on purpose?

Reply:
Good catch: The rule I was using looked for a character that wasn't a `.` after `etc`, it fails for `there is no character after etc`.